package.json

```diff
@@ -21,9 +21,9 @@
   "dependencies": {
     "broccoli-funnel": "^1.0.0 || ^2.0.0",
     "broccoli-merge-trees": "^1.0.0 || ^2.0.0",
-    "ember-cli-babel": "^6.6.0",
+    "ember-cli-babel": "^6.7.1",
```

Comment:
I don't understand why this is needed after `yarn upgrade pixi.js@~4.4.4` as that shouldn't touch anything other than `pixi.js` (and maybe dependencies of that which are unlikely to include anything that has a dependency on a particular `ember-cli-babel` version?).

Without this though I'm seeing:

```
ember-cli-htmlbars-inline-precompile v1.0.0 and above require the ember-cli-babel v6.7.1 or above. To use ember-cli-babel v5.x please downgrade ember-cli-htmlbars-inline-precompile to v0.3.
An error occured in the constructor for ember-cli-htmlbars-inline-precompile at /Users/marcoow/Documents/Code/ember-cli-pixijs/node_modules/ember-cli-htmlbars-inline-precompile
```

Reply:
my guess is that updating the pixijs dep also updated the `ember-cli-htmlbars-inline-precompile` dep within the current constraints and caused that warning message to disappear. #59 fixes this independently from this PR.

Reply:
Yeah but shouldn't `yarn upgrade <package>` not leave everything that doesn't need to be modified to upgrade that particular package unchanged?

Reply:
yeah, unless the `yarn.lock` was inconsistent before and you don't use `--pure-lockfile` or something like that